swarm/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # 0.42.0 [unreleased]
 
 - Remove uncontructed variant `Timer` from `ConnectionHandlerUpgrErr`.
+  Make `ListenUpgradeError::error` an `UpgradeError` instead of `ConnectionHandlerUpgrErr`, in case of timeout expiration we
+  just log the error as we don't know which ConnectionHandler should handle the error.
```

Comment:
These need a reference to this PR.

Reply:
yeah, it was done on 11db523b7d094f6ed1a73d32d332d534525c95db